src/services/ApiCalls.tsx

```diff
@@ -0,0 +1,63 @@
+import { URL_SERVER } from "constants/serverConnexion";
+
+const calculateAge = (birthday: any) => {
```

Comment:
You can specify the type for birthday (date, time or something like that, not sure what it is in ts)

Reply:
Done (Date type)